fraud/helpers.go

```diff
@@ -2,19 +2,30 @@ package fraud
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 )
 
-func join(p *pubsub.PubSub, proofType ProofType,
+func pubSubTopicID(fraudType, protocolSuffix string) string {
+	return "/fraud-sub/" + fraudType + "/v0.0.1/" + protocolSuffix
```

Comment:
so all fraudsubs will be on same version all the time?

Reply:
For now yes.

Reply:
Good point. Will do it. Thanks

Reply:
https://github.com/celestiaorg/celestia-node/issues/1517